python/setup.py

```diff
@@ -77,6 +77,7 @@ def build_extension(self, ext):
             os.makedirs(self.build_temp)
         subprocess.check_call(['cmake', ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
         subprocess.check_call(['cmake', '--build', '.'] + build_args, cwd=self.build_temp, env=env)
+        subprocess.check_call(['cmake', '--build', '.', '--target', 'install'] + build_args, cwd=self.build_temp, env=env)
```

Comment:
In very recent cmake versions (>= 3.15) this can be replaced by `cmake --install .`. I probably would not want to force using such a recent cmake version just for this line.